src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/Models/Dtos/AccountContactDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Umbraco.Forms.Integrations.Crm.ActiveCampaign.Models.Dtos
+{
+    public class AccountContactDto
+    {
+        [JsonPropertyName("id")]
+        public string Id { get; set; }
```

Comment:
Could this/should this be an `int`?  I just noticed you were parsing it to an `int` for one of the calls.